packages/quantic/docs/template/lwc-json/publish.js

```diff
@@ -104,9 +103,21 @@ function getMetadata(element) {
   return JSON.parse(xmlToJson(xmlData));
 }
 
+function getComponentCategory(element) {
+  return element.tags?.find((tag) => tag.originalTitle === 'category').value;
```

Comment:
It would fail when the tag list does not contain `category` because the `find` function would return `undefined`.

```suggestion
  return element.tags?.find((tag) => tag.originalTitle === 'category')?.value;
```